Jenkinsfile-testing

```diff
@@ -28,6 +28,7 @@ pipeline {
         checkout scm
         dir("npm") {
           sh "npm install"
+          sh "npx eslint src/**/*.ts"
```

Comment:
See [commit message](https://github.com/nationalarchives/tdr-transfer-frontend/pull/168/commits/75852b9250c48b9c8c6fcc658ea21c3100f2ab86) for an explanation.